integrations/supabase/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@trigger.dev/supabase",
+  "version": "0.1.0",
+  "description": "Trigger.dev integration for @supabase/supabase-js",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist/index.js",
+    "dist/index.d.ts",
+    "dist/index.js.map"
+  ],
+  "devDependencies": {
+    "@types/node": "18.x",
+    "rimraf": "^3.0.2",
+    "tsup": "7.1.x",
+    "typescript": "4.9.4"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "npm run clean && npm run build:tsup",
+    "build:tsup": "tsup",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@supabase/supabase-js": "^2.26.0",
+    "@trigger.dev/integration-kit": "workspace:^2.0.0-next.5",
+    "@trigger.dev/sdk": "workspace:^2.0.0-next.16",
+    "supabase-management-js": "^0.1.2",
+    "zod": "3.21.4"
+  },
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
Do we need to reduce all the package node requirements so they're inline with the SDK? 16 I think. Not urgent.

Reply:
The supabase-management-js package requires `>=18.0.0`, hence the reason this one does as well